pom.xml

```diff
@@ -228,7 +53,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.12</version>
+      <version>${junit.version}</version>
       <scope>test</scope>
     </dependency>
     <dependency>
```

Comment:
I assume that all dependencies below are specific to spoon-core and useless for spoon-decompiler, am I right?

Reply:
Yes. But if one turns out to be needed for `spoon-decompiler` too, we'll move it to `spoon-pom`, if that's what you mean.